libs/ledger-live-common/src/config/sharedConfig.ts

```diff
@@ -0,0 +1,78 @@
+import { ConfigSchema } from "@ledgerhq/live-config/LiveConfig";
+
+const liveConfig: ConfigSchema = {
```

Comment:
We could divide this config in multiple types, for instance this schema could be CosmosConfig and we extend it here ? just an idea

So cosmos logic stays in cosmos folder

Reply:
as this config variable "liveConfig" here is a global variable for all the config in ledger live. (It contains not only the cosmos family config, but other configs in the future)
So I type it as ConfigSchema. 

to inject this, we do this when the app starts:
LiveConfig.setConfig(liveConfig);

if we want to get a strong typed config as CosmosConfig.
we can do that on client side like this:
const cosmosConfig =  LiveConfig.getValueBykey("cosmos") as CosmosConfig;

if we move this config to the cosmos folder, the common part (where we did "LiveConfig.setConfig(liveConfig);") will refer to the cosmos part(inside the families folder) which is an anti pattern.

Reply:
import ConfigCosmos from 'families/cosmos/config.ts"
ConfigLLC: {
cosmos: ConfigCosmos
}

Reply:
fixed in my another PR:
https://github.com/LedgerHQ/ledger-live/pull/5763/files#diff-32a5530819ada406eb6bf26e05e32b0e3cd028eec5d59615726233eeebccb3b8R6

Reply:
Ok